users/views.py

```diff
@@ -13,7 +13,14 @@
 from rest_framework.permissions import AllowAny
 from rest_framework.throttling import AnonRateThrottle
 from rest_framework_simplejwt.views import TokenObtainPairView
+from .serializers import CustomTokenObtainPairSerializer
+import requests
 
+from django.contrib.auth import get_user_model
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
```

Comment:
If these are used only for the new OAuth view, consider grouping related imports for readability.